qcodes/dataset/data_set.py

```diff
@@ -262,6 +264,24 @@ def table_name(self):
     def guid(self):
         return get_guid_from_run_id(self.conn, self.run_id)
 
+    @property
+    def snapshot(self):
+        """Snapshot of the run as dictionary (or None)"""
+        snapshot_json = self.snapshot_raw
+        if snapshot_json is not None:
+            return json.loads(snapshot_json)
+        else:
+            return None
+
+    @property
+    def snapshot_raw(self):
+        """Snapshot of the run in JSON format (or None)"""
```

Comment:
Snapshot of the run as a string in JSON format (or None) ?

Reply:
yes. the `snapshot` property return python dict.

or is the formulation confusing or not good english?

Reply:
It's probably very minor and perhaps irrelevant now that the function has a type but I would write `a string in JSON format` to make it clear what the type being returned is